### PR TITLE
[Devtool]Add blob account preparer to enable versioning/lat/softdelete

### PR DESCRIPTION
**Pull request by xiafu-msft**

### CODE REVIEW

File: tools/azure-sdk-tools/devtools_testutils/storage_testcase.py [hunk 1]
```diff
@@ -137,4 +141,44 @@ def _get_resource_group(self, **kwargs):
...
+        if self.container_retention_days:
+            props.container_delete_retention_policy = DeleteRetentionPolicy(enabled=True,
+                                                                            days=self.container_retention_days)
+        if self.is_last_access_time_enabled:
+            props.last_access_time_tracking_policy = LastAccessTimeTrackingPolicy(enable=True)
+
+        if not all(prop is None for prop in props.as_dict().values()):
+            self.client.blob_services.set_service_properties(resource_group_name, account_name, props)
+
+        sleep(30)
```
Comment (seankane-msft) [inline tools/azure-sdk-tools/devtools_testutils/storage_testcase.py:179]: Is this sleep needed? If so, why?

Comment (xiafu-msft) [reply tools/azure-sdk-tools/devtools_testutils/storage_testcase.py:179]: it looks like those service properties need some time to take effect, though the time varies, 30s is fine for now

**Review by seankane-msft — APPROVED** [review]

**Comment by seankane-msft** [comment]
I don't see an issue, asking @scbedd for a second opinion

**Comment by xiafu-msft** [comment]
> I don't see an issue, asking @scbedd for a second opinion

sorry I just see this comment I will check with scott offline

**Pull request merged**